setup.cfg

```diff
@@ -36,7 +36,7 @@ install_requires =
     pandas~=2.0.0
     numpy~=1.21
     setuptools~=70.0.0
-    synapseclient~=4.0.0
+    synapseclient~=4.5.0
```

Comment:
4.6.0 is available if you wanted to use it.

Reply:
See PR description. `agora-data-tools` still supports Python 3.8.

Reply:
Gotcha. We might want to go back to 4.4.1 then. IIRC 4.5.x was the version that broke downloads of google buckets.

Reply:
Ah right. That functionality isn't currently used in ADT but will do